bridge_adaptivity/bridge_lti/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.conf.urls import url
 
 from bridge_lti.consumer import source_preview
-from bridge_lti.provider import lti_launch
+from bridge_lti.provider import lti_launch, create_session
```

Comment:
I think we could propose student to visit index bridge's page for new session creation.

Reply:
Index page also works too - My reasoning for making this new view was so that learners didn't get confused by seeing the login screen for the bridge when they went to the index page.